dev/com.ibm.ws.org.apache.cxf.cxf.core.3.2/src/org/apache/cxf/bus/osgi/CXFExtensionBundleListener.java

```diff
@@ -144,23 +143,39 @@ public void setServiceObject(Object o) {
             obj = o;
         }
         public Object load(ClassLoader cl, Bus b) {
-            if (interfaceName == null && bundle.getBundleContext() != null) {
-                ServiceReference<?> ref = bundle.getBundleContext().getServiceReference(className);
-                if (ref != null && ref.getBundle().getBundleId() == bundle.getBundleId()) {
-                    Object o = bundle.getBundleContext().getService(ref);
-                    serviceObject = o;
-                    obj = o;
-                    return obj;
+
+            // Liberty Change Start
```

Comment:
Since this is only for Java2 Security you should gate this in a `if (java2SecurityEnabled) {` block.  See #23113 for an example.